Resources/Private/Fusion/Prototypes/Checkbox.fusion

```diff
@@ -1,12 +1,31 @@
 prototype(Neos.Fusion.Form:Checkbox)  < prototype(Neos.Fusion.Form:Field) {
-    @context.field = ${Form.createFieldDefinition(this.form, this.name, this.property, null)}
+    @context.__multiple = ${this.multiple}
+
+    @propTypes {
+        multiple = ${PropTypes.boolean}
+        checked = ${PropTypes.boolean}
+    }
+
+    multiple = false
+    checked = false
+    # override checked state for object/submitted values
+    checked.@process.single = ${field.current == field.value}
```

Comment:
hm, do we want to name the properties "currentValue" and "initialValue" maybe?

Reply:
hm... I still struggle with finding good names...

Reply:
removed the given value from afx entirely now, `field`-context only deals with bound or submitted values. If a concrete fields has a value prop it is fine to use it.